aa_stripe/models.py

```diff
@@ -479,6 +479,18 @@ def refund(self, amount_to_refund=None):
                                                  charge=self.stripe_charge_id, amount=amount_to_refund)
             refund_id = stripe_refund["id"]
         except stripe.error.InvalidRequestError as e:
+            # it is possible some of the charge was refunded manually, refresh and try again
+            if e.code == "amount_too_large":
+                stripe_charge = stripe.Charge.retrieve(self.stripe_charge_id)
+                if stripe_charge.amount_refunded != self.amount_refunded:
+                    self.amount_refunded = stripe_charge.amount_refunded
+
+                # retry if we're not already retrying (prevent infinite loop)
+                if retry_on_error:
+                    return self.refund(amount_to_refund=None, retry_on_error=False)
```

Comment:
amount_to_refund=amount_to_refund? Or it is intentionally equal to None (which makes sense but would be good to explain it with comment)